tests/test_dropbox.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from blackbox.exceptions import MissingFields
+from blackbox.handlers.storage import Dropbox
+
+
+@pytest.fixture
+def mock_valid_dropbox_config():
+    return {"access_token": "XXXXXXX", "upload_directory": "/home/dropbox_user/Documents/"}
+
+
+@pytest.fixture
+def mock_invalid_dropbox_config():
+    return {}
```

Comment:
It's quite verbose making a fixture for an empty dictionary. Since the docstring explains the purpose of test, there's no need to _explain it with code_ as well 😄

Reply:
It happens a few times throughout the tests, maybe there's a specific reason you want these that I don't see?

Reply:
No reason at all other than I just decided to, I'm not familiar with tests and when to use what. I can definitely drop this and hardcode the empty dict in the tests if it improves the tests.

Reply:
It makes no functional difference, so it's fine either way :)